src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface CommentListProps {
+  comments: Comment[] | null;
+}
+
+export const CommentList: React.FC <CommentListProps> = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      { (!comments || comments.length === 0)
+        ? <b data-cy="NoCommentsMessage">No comments yet</b>
```

Comment:
Move this message to the `PostInfo`. This message will never render according to this condition 
<img width="387" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts/assets/99403198/eb01d83b-d86a-44e5-916e-13720e04edb6">

<img width="1540" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts/assets/99403198/e56f89df-f70d-4fda-a81f-627d45f25124">